virtio-devices/src/vhost_user/vu_common_ctrl.rs

```diff
@@ -135,10 +135,8 @@ pub fn setup_vhost_user<S: VhostUserMasterReqHandler>(
             inflight.info = info;
             inflight.fd = Some(fd);
         }
-        if let Some(inflight_fd) = inflight.fd.as_ref() {
-            vu.set_inflight_fd(&inflight.info, inflight_fd.as_raw_fd())
-                .map_err(Error::VhostUserSetInflight)?;
-        }
+        // Unwrapping the inflight fd is safe here since we know it can't be None.
+        vu.set_inflight_fd(&inflight.info, inflight.fd.as_ref.unwrap().as_raw_fd()).map_err(Error::VhostUserSetInflight)?;
```

Comment:
Haha that's fine you can squash it into the first commit. No need to mention my name :)

Reply:
Thanks, I also think we better squash it. 😀